consoleapp.py

```diff
@@ -5,6 +5,8 @@
 
 import re
 from consolestyle import fclr, style
+import readline
```

Comment:
Can't install module on Windows:
```
error: this module is not meant to work on Windows
```
Wrap this in a `try: ... except: pass` field.

Reply:
Interesting. I didn't get this error.
Now it is fixed. Does it work?

Reply:
(Note: you can also suggest in github when you are reviewing. If you suggest a thing I can accept it and commit it with a few clicks.)